test/e2e/framework/alertmanager.go

```diff
@@ -53,29 +53,38 @@ func (f *Framework) MakeBasicAlertmanager(name string, replicas int32) *v1alpha1
 	}
 }
 
-func (f *Framework) MakeAlertmanagerService(name, group string) *v1.Service {
-	return &v1.Service{
+func (f *Framework) MakeAlertmanagerNodePortService(name, group string) *v1.Service {
+	return f.MakeAlertmanagerService(name, group, v1.ServiceTypeNodePort)
+}
```

Comment:
How about making this accept the actual node port as an argument, calling `MakeAlertmanagerService` and setting the specified node port on the returned service.

This would make it clearer in the test why we are actually checking for `30903`.